src/helpers/swagger.ts

```diff
@@ -12,11 +12,11 @@ const swaggerOptions: Options = {
     },
     servers: [
       {
-        url: `http://localhost:${port}`,
+        url: `http://localhost:${port}/`,
         description: 'Local Server',
       },
       {
-        url: `https://e-commerce-bitcrafters-bn-1mpf.onrender.com`,
+        url: `https://e-commerce-bitcrafters-bn-1mpf.onrender.com/`,
         description: 'Deployed Server',
       },
```

Comment:
```suggestion
        url: `https://e-commerce-bitcrafters-bn-1mpf.onrender.com/`,
        description: 'Deployed Server',
      },
      {
          url: `/`,
        description: 'Current Server',
      },
```

Reply:
but this would better it would better if do it as fix

Reply:
Okay let me work on it

Reply:
This was in #55 , if possible we can revert back previous change